### PR TITLE
New testing section for onboarding road map.

### PR DESCRIPTION
**Pull request by IvailoStoyanov**
Because our experience with testing was quite mixed and unclear at times, me and @muelle2k  (Patrick) decided to try to improve the testing experience and make the roadmap more self-explanatory.

### CODE REVIEW

File: src/07-Testing.mdx [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+name: 07. Testing
+---
+
+import Authors from './helpers/Authors';
+import sloth from './assets/sloth.jpg'
+
+# Testing: Basic Unit Testing
+Level: *Useful Sloth*
```
Comment (jurekbarth) [inline src/07-Testing.mdx]: Can you make the level name an alliteration?

Comment (IvailoStoyanov) [reply src/07-Testing.mdx]: Subsidiary Sloth
Serviceable Sloth

Can't think of anything else...

Comment (jurekbarth) [reply src/07-Testing.mdx]: I’m fine with both go ahead and choose one 🙌🏻

**Comment by dweiger** [comment]
@IvailoStoyanov please resolve the conflicts. We moved to docsify in the meantime, so your branch is not up to date anymore.

**Pull request merged**